js-pkg/packages/driftdb-react/src/index.tsx

```diff
@@ -315,6 +316,47 @@ export function useLatency(): number | null {
   return latency
 }
 
+type DataChannelMsg = { sender: string; value: any }
+type WebRtcOnMessage = (msg: DataChannelMsg) => void
+
+function useWebRtcBroadcastChannel(throttle = 0) {
+  const db = useDatabase()
+  const id = useUniqueClientId()
+  const WebRtcBroadcastChannelRef = React.useRef<SyncedWebRTCConnections>()
+  if (!WebRtcBroadcastChannelRef.current) {
+    WebRtcBroadcastChannelRef.current = new SyncedWebRTCConnections(db, id, throttle)
+  }
+  return {
+    send: (msg: string) => WebRtcBroadcastChannelRef.current!.send(msg),
+    setOnMessage: (onMessage: WebRtcOnMessage) =>
+      WebRtcBroadcastChannelRef.current!.setOnMessage(onMessage),
+    peers: WebRtcBroadcastChannelRef.current!.peers()
+  }
+}
+
+export function useWebRtcPresence(vals: any, throttle = 0) {
```

Comment:
Since this is part of the public API, let's add an explicit return type along with a docstring comment.